README.rst

```diff
@@ -84,6 +84,13 @@ If you use structlog, add the following configuration setting to your INI file t
     pyramid_heroku.structlog = true
 
 
+If you are serving the app behind reverse proxy, add the following
+configuration setting to your INI file to have the correct client IP set in
+``ClientAddr`` tween::
+
+    pyramid_heroku.reverse_proxy = true
```

Comment:
> If tween would only check for presence of headers one could spoof IP by providing `CF` headers and `X-Forwarded-For` header.

That would only happen if CF-Proxy is removed from the chain. https://support.cloudflare.com/hc/en-us/articles/200170786

Reply:
I was trying to say that only checking for `CF` headers would make it possible to spoof IP for apps that don't use Cloudflare by faking `CF` headers.

 The safer way is then to have some configuration option. It then makes sense to make it general instead of Cloudflare specific.

Reply:
> For our use case, we can just remove any Cloudflare IPs from the X-Forwarded-For list of IPs, and that's it. Secure, and no configuration needed.

The only thing I don't like about this is maintaining the list. It would be hard to debug that geolocation on minisites isn't working on some cases due to outdated Cloudflare IPs here.

Reply:
The list is downloaded at pyramid app start. With a timeout of 10 seconds and fallback to an empty list.